File: online_judge_solutions/uri_2029.py
# Abelhas - URI 2029

while True:
	try:
		v = float(raw_input())
		d = float(raw_input())
		r = d / 2

		area = 3.14 * (r * r)
		h = v / area

		print "ALTURA = %.2f" %h
		print "AREA = %.2f" %area

	except: break

File: online_judge_solutions/uri2029.cpp
//mel e seu joao - uri 2029

#include <bits/stdc++.h>

using namespace std;

int main(){
	double vol, d;
	cin >> vol;
	cin >> d;
	
	while(!EOFError){
		double r = d / 2;
		double a = 3.14 * r * r;
		double h = vol / a;
		
		cout << "ALTURA = " << h << endl;
		cout << "AREA = " << a << endl;
		
		cin >> vol;
		cin >> d;
	}
	
	return 0;
}
